crates/misc/tct-visualize/Cargo.toml

```diff
@@ -21,7 +21,9 @@ tokio-util = {workspace = true, features = ["full"]}
 tonic = {workspace = true}
 prost = {workspace = true}
 tokio-stream = {workspace = true}
-axum = {workspace = true, features = ["headers", "query"]}
+axum = {workspace = true }
+# TODO: consider using the Query extractor provided by axum-extra
```

Comment:
The query extractor is now provided by both axum and axum-extra, although they have minute differences. Just flagging this here.